package.json

```diff
@@ -48,12 +48,13 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.11.2",
-    "@salesforce/scv-connector-base": "^3.1.51",
+    "@salesforce/scv-connector-base": "file:../scv-connector-base",
     "axios": "^0.21.1",
     "custom-env": "^2.0.1",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
-    "nodemon": "^2.0.4"
+    "nodemon": "^2.0.4",
+    "scv-connector-base": "3.1.77"
```

Comment:
@dlouvton Need to remove this and replace the local reference above with the version